src/codeenginestudio/thuy/Exercise2.java

```diff
@@ -0,0 +1,36 @@
+package codeenginestudio.thuy;
+
+public class Exercise2 {
+    public static void main(String[] args) {
+        String input = "This is another test";
+        String[] araySlipted = splitString(input);
+        String ouput = convertArraytoString(checkLengthtReverseString(araySlipted));
+        System.out.println("ouput: " + ouput);
+    }
+
+    private static String convertArraytoString(String[] StringArray) {
+        String stringConverted = String.join(" ", StringArray);
+        return stringConverted;
+    }
+
+    private static String[] checkLengthtReverseString(String[] stringArray) {
+
+        for (int i = 0 ; i < stringArray.length; i++) {
+            if(stringArray[i].length() >= 5) {
```

Comment:
Please make "5" to be a constant value.

Reply:
it means I need to create a parameter to change "5" or other number

Reply:
yes, may be you can use `private static constants Integer MAX_LENGTH = 5;`